react/BankInvoiceSection.tsx

```diff
@@ -24,10 +24,17 @@ const messages = defineMessages({
 const BankInvoiceSection: FC = () => {
   const handles = useCssHandles(CSS_HANDLES)
   const orderGroup = useOrderGroup()
-  const paymentInfo = getPaymentInfoFromOrder(orderGroup.orders[0])
   const { formatMessage } = useIntl()
 
-  const { url, paymentSystemName, barCodeNumber } = paymentInfo
+  const paymentMethodsFromOrder = getPaymentMethodsInfoFromOrder(
+    orderGroup.orders[0]
+  )
+  const bankInvoice = paymentMethodsFromOrder
+    .flat()
+    .find(({ paymentGroup, url }) => paymentGroup === 'bankInvoice' && !!url)
```

Comment:
Here the intention is to prioritize the bank invoice?

Reply:
Not really, the main goal here is to find any bank invoice being used as a payment method